bin/server_package_debian

```diff
@@ -20,6 +20,7 @@
 OS_DIR_BIN="/usr/bin"
 OS_DIR_CONFIG="/etc/mysterium-node"
 OS_DIR_INSTALLATION="/usr/lib/mysterium-node/installation"
+BINARY=${BINARY:="build/client/mysterium_server"}
```

Comment:
should be "build/server/mysterium_server"

Reply:
Fixed mistype